core_backend/app/db/db_models.py

```diff
@@ -294,3 +309,196 @@ async def save_waresponse_to_db(
     await asession.commit()
     await asession.refresh(wa_response)
     return wa_response
+
+
+class ContentDB(Base):
+    """
+    SQL Alchemy data model for content
+    """
+
+    __tablename__ = "content"
+
+    content_id: Mapped[int] = mapped_column(Integer, primary_key=True, nullable=False)
+    content_embedding: Mapped[Vector] = mapped_column(
+        Vector(int(PGVECTOR_VECTOR_SIZE)), nullable=False
+    )
+    content_title: Mapped[str] = mapped_column(String(length=150), nullable=False)
+    content_text: Mapped[str] = mapped_column(String(length=1446), nullable=False)
```

Comment:
Here the length is at 1446

Reply:
Yes but ContentDB doesn't do any check. It is just a reference to get the migrations. Meaning that the constraint is on the DB level not on the code level. So if we add data to the DB using ContentDB. The code won't check the character length but the query will fail on postgres

Reply:
The best way to address that issue is to set the max character to a higher value. Is the current value a hard limit? Otherwise we can limit the text of each FAQ to be 1446 in the validation script.

Reply:
I will increase the limit to 2000. We need to figure out how to handle communicating with whatsapp because I understand whatsapp have a max character limit